scripts/unsupported/combine_tracks_postprocessing_cnv/combine_tracks.wdl

```diff
@@ -618,4 +644,53 @@ task PrepareForACSConversion {
     output {
         File model_and_calls_merged_gatk_seg = "${output_name}.final.seg"
     }
+}
+
+# Must be python3 in the docker image
+task Gistic2Convert {
+    File input_file
+    String docker
+    String output_file = basename(input_file) + ".gistic2.seg"
+
+    command <<<
+        set -e
+        python <<EOF
+import csv
```

Comment:
white space?  (not sure if this is intentional)

Reply:
This was intentional.  No action.